YellowNotes.Core/Dtos/UserDto.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace YellowNotes.Core.Dtos
+{
+    public class UserDto
+    {
+        [Required]
```

Comment:
Może jakiś Error Message, np
```suggestion
        [Required(ErrorMessage = "Email is required!")]
```

Reply:
Imo mało to potrzebne, bo i tak mamy walidację po stronie aplikacji. Jak uważasz ☺️.

Reply:
może się przydać przy debugu, zamiast dostać "error" dostaniesz info co i jak, ale jak uważasz że niepotrzebne, to spoko